save-backend/build.gradle.kts

```
@@ -8,13 +8,18 @@ import org.jetbrains.kotlin.gradle.tasks.KotlinCompile
 plugins {
     kotlin("jvm")
     // this plugin will generate generateOpenApiDocs task
-    // running this task, it will writes the OpenAPI spec into a backend-api-docs.json file in save-backend dir.
+    // running this task, it will write the OpenAPI spec into a backend-api-docs.json file in save-backend dir.
     id("org.springdoc.openapi-gradle-plugin") version "1.3.4"
 }
 
 openApi {
     apiDocsUrl.set("http://localhost:5800/internal/v3/api-docs/latest")
+    outputDir.set(file(rootDir))
```

```suggestion
    outputDir.set(file(projectDir))
```
Did you try it like this?

---

I tried, but for some reason, this didn't worked, it writes, that there is no any `backend-api-docs.json` file, even if it exist and failed with this error

---

What about combination of `outputDir.set(file(projectDir))` and `outputFileName.set("backend-api-docs.json")`? Since the plugin generates full file name relative to `outputDir` (as we've seen from source) there is no need to set `outputFileName` as an absolute path

---

Seems, that it's works